test/setup/test_multiple_storage_sets.py

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env python3
+
+# CORTX Python common library.
+# Copyright (c) 2021 Seagate Technology LLC and/or its Affiliates
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+# For any questions about this software or licensing,
+# please email opensource@seagate.com or cortx-questions@seagate.com.
+
+import os
+import yaml
+import unittest
+from cortx.provisioner.provisioner import CortxProvisioner
+
+cortx_conf_url = "yaml:///tmp/test_conf_store.conf"
+
+
+class TestMultipleStorageSet(unittest.TestCase):
+    """ Test multiple storage_sets """
+
+    def test_for_multiple_storage_sets(self):
+        """ Test for multiple storage sets """
+
+        cluster_conf_file_path = "/tmp/invalid_cluster.yaml"
```

Comment:
so what is invalid in this config ?

Reply:
No there is nothing invalid in config for this test, it has multiple storage sets.
Changed file name to cluster.yaml.